ui2/src/common/Table.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Table } from 'antd';
+import _ from 'lodash';
+import './Table.css';
+
+export const AppTable = ({ columns, filteredItems, onItemSelectAll, onItemSelect, selectedKeys, scroll }) => {
```

Comment:
also here you can spread the rest if you going to pass the to the component

Reply:
done